README.md

```diff
@@ -46,6 +46,8 @@ For example, on macOS with [Homebrew](http://brew.sh/):
 brew install cmake pkg-config icu4c
 ```
 
+If you are on macOS and run `script/bootstrap` in this repository, those dependencies will be installed for you provided Homebrew is available.
```

Comment:
I don't think we should mention this here as we don't want or need peeps installing just the gem to run `script/bootstrap` as this will download all the grammar submodules too.

Lets drop this line for now.

```suggestion
```